app/scripts/modules/tencentcloud/src/securityGroup/details/index.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
```

Comment:
`index.ts` files should contain only re-exports, not components.  Move this code to `SecurityGroupDetail.tsx`